clangd-tidy

```diff
@@ -234,6 +311,13 @@ if __name__ == "__main__":
     parser.add_argument(
         "-v", "--verbose", action="store_true", help="Print verbose output from clangd."
     )
+    parser.add_argument(
+        "-c",
+        "--context",
+        type=int,
+        default=2,
+        help="Number of lines of code shown either side of the detected issue. [default: 2]",
```

Comment:
I don't quite understand this help message.

Reply:
This is provides how much code is shown around an error. For example when its the default 2 you might get:
```bash
/home/user/west-workspace/notus/subsys/file_system/file_system.cpp:22:26: Error: Variable name 'a' is too short, expected at least 3 characters [readability-identifier-length]
   20 |      arch_nop();
   21 |  
   22 |      [[maybe_unused]] int a = 0;
      |                           ^
   23 |  }
ninja: build stopped: subcommand failed.
```

when it's 5 you could get:
```bash
/home/user/west-workspace/notus/subsys/file_system/file_system.cpp:22:26: Error: Variable name 'a' is too short, expected at least 3 characters [readability-identifier-length]
   17 |  
   18 |  FileSystem::~FileSystem()
   19 |  {
   20 |      arch_nop();
   21 |  
   22 |      [[maybe_unused]] int a = 0;
      |                           ^
   23 |  }
   24 |  
   25 |  void other_fun()
   26 |  {
   27 |
```

Happy to update the help message if you have any ideas